### ISSUE TITLE
SSL pinning IOS

### ISSUE THREAD

**Issue by Loukious** [Tustin/fortnite-php#63]
So now IOS got an update and it now has SSL pinning enabled. Could you provide us with a patch?

**Comment by mkelzeer** [comment]
Did you get a solution?

**Comment by Loukious** [comment]
> Did you get a solution?

Nope. But I managed to get the endpoints that I needed through game logs.
